performance_manager/lib/postgres_utils.py

```diff
@@ -147,6 +147,27 @@ class VehiclePositionEvents(SqlBase):  # pylint: disable=too-few-public-methods
     )
 
 
+class TripUpdateEvents(SqlBase):  # pylint: disable=too-few-public-methods
+    """Table for GTFS-RT Trip Update Predicted Stop Events"""
+
+    __tablename__ = "eventsTripUpdates"
```

Comment:
add trip updates schema

Reply:
i kinda feel like we should move the sql base stuff to its own file. schema stuff in one place, utilities in another.

something to keep in mind moving forward.